src/app/_components/FilterContainer.tsx

```diff
@@ -62,6 +62,18 @@ function ResultsAndCategory({ results, category }: ResultsAndCategoryProps) {
   )
 }
 
+function ResultsAndCategoryCompact({
+  results,
+  category,
+}: ResultsAndCategoryProps) {
+  return (
+    <div className="hidden flex-col gap-8 lg:flex">
+      {results}
+      {category}
+    </div>
+  )
+}
+
```

Comment:
Created a new component that has a smaller gap than ResultsAndCategory. See below how it looks with gap-10. Alternatively we could pass a prop, but thought this looks cleaner in the code base.

<img width="199" alt="Screenshot 2024-08-13 at 17 34 34" src="https://github.com/user-attachments/assets/4614ae8e-91e7-4bce-aa48-7ade5e51a949">

Reply:
Fine by me :)